### DISCUSSION TITLE
How to ignore duplicated link? or ignore image link?

### DISCUSSION THREAD

**Discussion by av930** [gdh1995/vimium-c#881] [Q&A]
I usually use Jenkins on my work. so I'd like to control it with vimium-C, however there is difficulty that so many duplicated link is showed up like this
![image](https://user-images.githubusercontent.com/990793/223929716-a3ff3ac0-9a61-480d-9fc0-5486de6894a0.png)

url of left image icon is same to text url.

Therfore it makes me crazy there is so many shortcut tips.
![image](https://user-images.githubusercontent.com/990793/223929850-33f3f7f2-fdc5-4258-b37c-496b25d56028.png)

Is there any way to make it simple? only shows the one shortcut for same url?

**Comment by gdh1995** [comment]
You may write CSS-selectors to include and exclude elements by yourself: see https://github.com/gdh1995/vimium-c/wiki/Add-or-remove-hints
